dubbo native-image run failed.

Environment
 <dubbo.version>3.3.0-beta.1</dubbo.version>

This is Exception information:

```
2023-11-29T09:15:49.522+08:00  INFO 5058 --- [           main] d.s.b.c.e.WelcomeLogoApplicationListener :  [DUBBO] 

 :: Dubbo (v3.3.0-beta.1) : https://github.com/apache/dubbo
 :: Discuss group : dev@dubbo.apache.org
, dubbo version: 3.3.0-beta.1, current host: 192.168.0.219

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v3.0.0)

2023-11-29T09:15:49.528+08:00 ERROR 5058 --- [           main] o.s.boot.SpringApplication               : Application run failed

org.springframework.cglib.core.CodeGenerationException: java.lang.NoSuchMethodException-->org.apache.dubbo.spring.boot.autoconfigure.DubboRelaxedBinding2AutoConfiguration$$SpringCGLIB$$0.CGLIB$SET_THREAD_CALLBACKS([Lorg.springframework.cglib.proxy.Callback;)
	at org.springframework.cglib.proxy.Enhancer$EnhancerFactoryData.<init>(Enhancer.java:506)
	at org.springframework.cglib.proxy.Enhancer.wrapCachedClass(Enhancer.java:801)
	at org.springframework.cglib.core.AbstractClassGenerator$ClassLoaderData$3.apply(AbstractClassGenerator.java:114)
	at org.springframework.cglib.core.AbstractClassGenerator$ClassLoaderData$3.apply(AbstractClassGenerator.java:111)
	at org.springframework.cglib.core.internal.LoadingCache$2.call(LoadingCache.java:55)
	at java.base@17.0.5/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at org.springframework.cglib.core.internal.LoadingCache.createEntry(LoadingCache.java:62)
	at org.springframework.cglib.core.internal.LoadingCache.get(LoadingCache.java:35)
	at org.springframework.cglib.core.AbstractClassGenerator$ClassLoaderData.get(AbstractClassGenerator.java:137)
	at org.springframework.cglib.core.AbstractClassGenerator.create(AbstractClassGenerator.java:322)
	at org.springframework.cglib.proxy.Enhancer.createHelper(Enhancer.java:567)
	at org.springframework.cglib.proxy.Enhancer.createClass(Enhancer.java:412)
	at org.springframework.context.annotation.ConfigurationClassEnhancer.createClass(ConfigurationClassEnhancer.java:138)
	at org.springframework.context.annotation.ConfigurationClassEnhancer.enhance(ConfigurationClassEnhancer.java:109)
	at org.springframework.context.annotation.ConfigurationClassUtils.initializeConfigurationClass(ConfigurationClassUtils.java:81)
	at org.apache.dubbo.spring.boot.autoconfigure.DubboRelaxedBinding2AutoConfiguration__BeanDefinitions.getDubboRelaxedBindingAutoConfigurationBeanDefinition(DubboRelaxedBinding2AutoConfiguration__BeanDefinitions.java:24)
	at com.yuhoutian.demo.provider.ProviderApplication__BeanFactoryRegistrations.registerBeanDefinitions(ProviderApplication__BeanFactoryRegistrations.java:378)
	at com.yuhoutian.demo.provider.ProviderApplication__ApplicationContextInitializer.initialize(ProviderApplication__ApplicationContextInitializer.java:26)
	at com.yuhoutian.demo.provider.ProviderApplication__ApplicationContextInitializer.initialize(ProviderApplication__ApplicationContextInitializer.java:19)
	at org.springframework.context.aot.AotApplicationContextInitializer.initialize(AotApplicationContextInitializer.java:72)
	at org.springframework.context.aot.AotApplicationContextInitializer.lambda$forInitializerClasses$0(AotApplicationContextInitializer.java:61)
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:603)
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:383)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:307)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1302)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1291)
	at com.yuhoutian.demo.provider.ProviderApplication.main(ProviderApplication.java:33)
Caused by: java.lang.NoSuchMethodException: org.apache.dubbo.spring.boot.autoconfigure.DubboRelaxedBinding2AutoConfiguration$$SpringCGLIB$$0.CGLIB$SET_THREAD_CALLBACKS([Lorg.springframework.cglib.proxy.Callback;)
	at java.base@17.0.5/java.lang.Class.getDeclaredMethod(DynamicHub.java:2675)
	at org.springframework.cglib.proxy.Enhancer.getCallbacksSetter(Enhancer.java:907)
	at org.springframework.cglib.proxy.Enhancer$EnhancerFactoryData.<init>(Enhancer.java:495)
	... 26 common frames omitted

```

I have pulled you lastest code and can't reproduce,just got 

```
2023-11-30T18:05:14.233+08:00 ERROR 72720 --- [           main] o.s.boot.SpringApplication               : Application run failed

java.io.UncheckedIOException: java.io.FileNotFoundException: class path resource [META-INF/dubbo/default/actuator-endpoints.properties] cannot be opened because it does not exist
        at com.yuhoutian.demo.provider.ProviderApplication__ApplicationContextInitializer.processPropertySources(ProviderApplication__ApplicationContextInitializer.java:39)
        at com.yuhoutian.demo.provider.ProviderApplication__ApplicationContextInitializer.initialize(ProviderApplication__ApplicationContextInitializer.java:25)
        at com.yuhoutian.demo.provider.ProviderApplication__ApplicationContextInitializer.initialize(ProviderApplication__ApplicationContextInitializer.java:19)
        at org.springframework.context.aot.AotApplicationContextInitializer.initialize(AotApplicationContextInitializer.java:72)
        at org.springframework.context.aot.AotApplicationContextInitializer.lambda$forInitializerClasses$0(AotApplicationContextInitializer.java:61)
        at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:605)
        at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:385)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:309)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:1304)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:1293)
        at com.yuhoutian.demo.provider.ProviderApplication.main(ProviderApplication.java:33)
Caused by: java.io.FileNotFoundException: class path resource [META-INF/dubbo/default/actuator-endpoints.properties] cannot be opened because it does not exist
        at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:211)
        at org.springframework.core.io.support.EncodedResource.getInputStream(EncodedResource.java:160)
        at org.springframework.core.io.support.PropertiesLoaderUtils.fillProperties(PropertiesLoaderUtils.java:100)
        at org.springframework.core.io.support.PropertiesLoaderUtils.fillProperties(PropertiesLoaderUtils.java:74)
        at org.springframework.core.io.support.PropertiesLoaderUtils.loadProperties(PropertiesLoaderUtils.java:60)
        at org.springframework.core.io.support.ResourcePropertySource.<init>(ResourcePropertySource.java:67)
        at org.springframework.core.io.support.DefaultPropertySourceFactory.createPropertySource(DefaultPropertySourceFactory.java:37)
        at org.springframework.core.io.support.PropertySourceProcessor.processPropertySource(PropertySourceProcessor.java:87)
        at com.yuhoutian.demo.provider.ProviderApplication__ApplicationContextInitializer.processPropertySources(ProviderApplication__ApplicationContextInitializer.java:37)
        ... 10 common frames omitted

```

missing some config file, not the same error

The summary is as follows：

Q:  java.io.UncheckedIOException: java.io.FileNotFoundException: class path resource [META-INF/dubbo/default/actuator-endpoints.properties] cannot be opened because it does not exist
A: set src\main\resources\META-INF\native-image\resource-config.json

Q: Caused by: java.lang.ClassNotFoundException: com.alibaba.cloud.dubbo.registry.SpringCloudRegistryFactory
A: set src\main\resources\META-INF\native-image\reflect-config.json

Q: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'feignTargeter': Proxy class defined by interfaces [interface org.springframework.cloud.openfeign.Targeter] not found. Generating proxy classes at runtime is not supported. Proxy classes need to be defined at image build time by specifying the list of interfaces that they implement. To define proxy classes use -H:DynamicProxyConfigurationFiles=<comma-separated-config-files> and -H:DynamicProxyConfigurationResources=<comma-separated-config-resources> options.

A: set src\main\resources\META-INF\native-image\proxy-config.json

--- a/dubbo-spring-boot/dubbo-spring-boot-autoconfigure/src/main/java/org/apache/dubbo/spring/boot/autoconfigure/DubboRelaxedBinding2AutoConfiguration.java
+++ b/dubbo-spring-boot/dubbo-spring-boot-autoconfigure/src/main/java/org/apache/dubbo/spring/boot/autoconfigure/DubboRelaxedBinding2AutoConfiguration.java
@@ -51,7 +51,7 @@ import static org.springframework.beans.factory.config.ConfigurableBeanFactory.S
  * @see DubboRelaxedBindingAutoConfiguration
  * @since 2.7.0
  */
-@Configuration
+@Configuration(proxyBeanMethods = false)
 @ConditionalOnProperty(prefix = DUBBO_PREFIX, name = "enabled", matchIfMissing = true)
 @ConditionalOnClass(name = "org.springframework.boot.context.properties.bind.Binder")
 @AutoConfigureBefore(DubboRelaxedBindingAutoConfiguration.class)

--- a/dubbo-spring-boot/dubbo-spring-boot-compatible/autoconfigure/src/main/java/org/apache/dubbo/spring/boot/autoconfigure/DubboAutoConfiguration.java
+++ b/dubbo-spring-boot/dubbo-spring-boot-compatible/autoconfigure/src/main/java/org/apache/dubbo/spring/boot/autoconfigure/DubboAutoConfiguration.java
@@ -62,7 +62,7 @@ public class DubboAutoConfiguration {
     @ConditionalOnProperty(prefix = DUBBO_SCAN_PREFIX, name = BASE_PACKAGES_PROPERTY_NAME)
     @ConditionalOnBean(name = BASE_PACKAGES_BEAN_NAME)
     @Bean
-    public ServiceAnnotationPostProcessor serviceAnnotationBeanProcessor(
+    public static ServiceAnnotationPostProcessor serviceAnnotationBeanProcessor(
             @Qualifier(BASE_PACKAGES_BEAN_NAME) Set<String> packagesToScan) {
         ServiceAnnotationPostProcessor serviceAnnotationPostProcessor;
         try {

